src/transform.ts

```diff
@@ -9,6 +9,7 @@ const DEFAULT_OPTIONS = {
 		nativeClassProperties: true,
 		noEmptyExport: true,
 		importNotUsedAsValues: "preserve",
+		deprecatedTsModuleAsError: true,
```

Comment:
@magic-akari ptal

Reply:
The `deprecatedTsModuleAsError` option should be configured as a top-level setting alongside `mode` and `transform` (not nested within `transform`).

This is required because the option also needs to take effect in `strip-only` mode.